### DISCUSSION TITLE
Welcome to cryotomosim_CTS Discussions!

### DISCUSSION THREAD

**Discussion by carsonpurnell** [carsonpurnell/cryotomosim_CTS#1] [Announcements]
announcement:

test discussion to see if this works

**Comment by JaeEYang** [comment]
Hi there, I installed 0.6.8 and used a tutorial pdb file (6lfm_gprotein.membrane.cif). I selected the default number of vesicles (n = 3). I got the model that has g proteins placed towards the interior of the vesicles instead of sticking out towards the exterior. I wonder what the issue could possibly be? Many thanks! 
<img width="753" height="520" alt="CTSGUI_error" src="https://github.com/user-attachments/assets/f3459f7d-ca9f-4117-989a-ac2b0688d447" />

**Comment by carsonpurnell** [comment]
There is no issue, the GPCR complex is by mass mostly intercellular/within envelope, not externally protruding. A bare GPCR would indeed typically show the opposite.
